### ISSUE TITLE
TensorBoard Graph doesnt work

### ISSUE THREAD

**Issue by DaniPol** [keras-team/keras#16810]
win 10 
python 3.10.5
tensorflow 2.9.1

I ran this code:
![image](https://user-images.githubusercontent.com/73639212/179839398-e85a0f48-d4ac-4bc6-8fac-5bbd08bc173f.png)
A folder created:
![image](https://user-images.githubusercontent.com/73639212/179839514-7dfe24f3-e481-468d-9b28-37318b879d61.png)
I used this command in the terminal:
![image](https://user-images.githubusercontent.com/73639212/179839617-c6e4ee51-7862-4999-b10b-f6ffb1a3bf29.png)
and this happend:
![image](https://user-images.githubusercontent.com/73639212/179839754-852ec49f-290e-4e29-ac5d-52a0e3b62bce.png)
I clicked the link (http://localhost:6006/ ) and then graphs and it send me to this:
![image](https://user-images.githubusercontent.com/73639212/179839985-ab6f7612-badd-4165-9c54-e4ab4140513e.png)
I pressed Conceptual graph in the Graph type and got this:
![image](https://user-images.githubusercontent.com/73639212/179840145-779ddea1-b3f8-4d6a-b978-bcdc4a054651.png)
This is the output that i want:
![image](https://user-images.githubusercontent.com/73639212/179840277-d39fe6e3-3e0a-4cd3-8fab-f1974e40b04a.png)
What should I do?

**Comment by DaniPol** [comment]
import pandas as pd
import keras
from keras.models import Sequential
from keras.layers import *

RUN_NAME = "run 2 with 5 nodes"

training_data_df = pd.read_csv("sales_data_training_scaled.csv")

X = training_data_df.drop('total_earnings', axis=1).values
Y = training_data_df[['total_earnings']].values

# Define the model
model = Sequential()
model.add(Dense(5, input_dim=9, activation='relu', name='layer_1'))
model.add(Dense(100, activation='relu', name='layer_2'))
model.add(Dense(50, activation='relu', name='layer_3'))
model.add(Dense(1, activation='linear', name='output_layer'))
model.compile(loss='mean_squared_error', optimizer='adam')

# Create a TensorBoard logger
logger = keras.callbacks.TensorBoard(
    log_dir='logs/{}'.format(RUN_NAME),
    histogram_freq=5,
    write_graph=True
)

# Train the model
model.fit(
    X,
    Y,
    epochs=50,
    shuffle=True,
    verbose=2,
    callbacks=[logger]
)

# Load the separate test data set
test_data_df = pd.read_csv("sales_data_test_scaled.csv")

X_test = test_data_df.drop('total_earnings', axis=1).values
Y_test = test_data_df[['total_earnings']].values

test_error_rate = model.evaluate(X_test, Y_test, verbose=0)
print("The mean squared error (MSE) for the test data set is: {}".format(test_error_rate))``

**Comment by tilakrayal** [comment]
@DaniPol,
I was facing a different error while executing the provided code. Kindly find the gist of it [here](https://colab.research.google.com/gist/tilakrayal/35adc3c2ea43f19ced577c6485024b8e/untitled461.ipynb). Could you provide the dependencies and code in the indentation format. Thank you!
